ruby.rb

```diff
@@ -6,37 +6,66 @@
 
 # your ruby loop here
 
+tempArray.each do |element| 
+    puts element * 2
+ end
+ 
+ tempArray.map { |n| n * 2 }
+
 
 #### 2. From all the built in Ruby methods we've seen in class this week, choose three that you think are particularly helpful and create examples to show how they work.
 
 # your three built in ruby methods
 
 
+
```

Comment:
You are using the to_string and to_float built in methods in your example. Can you find one more?